collection-scripts/gather

```diff
@@ -41,9 +41,6 @@ done
 # Gather etcd information
 /usr/bin/gather_etcd
 
-# Collect System Audit Logs
-/usr/bin/gather_audit_logs
```

Comment:
you can't do this until the e2e test works.

Reply:
that e2e you're talking about already explicitly calls this script see https://github.com/openshift/origin/blob/1a83021f4dbc91de606a71d3a27cd80664aae26c/test/extended/cli/mustgather.go#L160 so it shouldn't block this PR